golang/lambda/example_lambda.go

```diff
@@ -0,0 +1,20 @@
+package lambda
+
+import (
+	"github.com/kurtosis-tech/kurtosis-client/golang/lib/networks"
+	"github.com/sirupsen/logrus"
+)
+
+type ExampleLambda struct {
+}
+
+func NewExampleLambda() *ExampleLambda {
+	return &ExampleLambda{}
+}
+
+func (e ExampleLambda) Execute(networkCtx *networks.NetworkContext, serializedParams string) (serializedResult string, resultError error) {
+	logrus.Infof("Example Lambda receives serializedParams '%v'", serializedParams)
```

Comment:
This execute method was only for test purposes, I want to implement something better, I don't know what, maybe a Lamba that returns the current time or a Kurtosis Text with random advice I don't know What do you think @mieubrisse ?

Reply:
Hahaha I love the idea of a text with random advice - "Brush your teeth every day" "Don't pet a burning dog" "Even a broken clock is right twice a day" 🤣 🤣